learn/how-tos/ssl-offloading.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "SSL Offloading in LoadBalancer"
+id: ""
+---
+---
+
+
+SSL offloading is the process of removing the SSL based encryption from incoming traffic that a web server receives to relieve it from decryption of data. 
+
+LoadBalancers will verify SSL and decrypt the data before sending to backend nodes.
+Backend nodes will always receive http trafic with few `X-Forwarded` headers. 
+
+WaveMaker Apps can be deployed in various Web/Application Servers. 
+We will discuss here what needs to be taken care while Deploying WaveMaker App into Tomcat Server. Similar approach needs to be followed for other Servers.
+
+- LoadBalancers will offload ssl and sends http trafic to tomcat.
+- LoadBalancers will add below Headers to tomcat server.
+   - `X-Forwarded-Proto`
+   - `X-Forwarded-For`
+   - `X-Forwarded-Port`
+- Our WaveMaker app checks the request with request.isSecure() to validate whether it is https or not.
+- Tomcat should able to read this headers and trust that its https.
+- Default tomcat installation don't read these headers. So we have configure Tomcat valve to read all these headers.
```

Comment:
replace it with "do not honor these headers".

Reply:
changed